source/query-with-natural-language/ai-and-data-usage-information.txt

```diff
@@ -0,0 +1,44 @@
+.. _compass-ai-data-usage:
+
+=============================
+AI and Data Usage Information
+=============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+Querying with natural language in Compass is powered by Generative AI
+(Gen AI), and may give inaccurate responses. Please see our `Generative AI FAQ 
+<https://dochub.mongodb.org/core/faq-ai-features>`__ 
+for more information about Gen AI in MongoDB products.
+
+Third Party Providers
+---------------------
+
+This feature currently uses the `Azure OpenAI Service
```

Comment:
(vale suggestion)
I like "now" over "currently". But up to you.

Reply:
I prefer currently in this as not to suggest the AI service has changed recently but is subject to change in future.